types/relationship.js/dist/relationship-mode.min.d.ts

```diff
@@ -1,6 +1,2 @@
-export as namespace relationshipMode;
-export = relationshipMode;
```

Comment:
When I checked the file last, it was a UMD files that declared a global. Has that changed?

Reply:
Thank you for pointing out the problem, I have corrected it